release-notes/SupportedFeatures-v1.0.0-rc2.md

```diff
@@ -21,38 +29,38 @@ Supported features
-||Https| [:warning:](#transports_https) | :white_check_mark: | [:warning:](#transports_https) | :grey_question: | :grey_question: | :grey_question: | [:warning:](#transports_https) |
-||Tcp|:white_check_mark: | :white_check_mark: | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | [:no_entry_sign:](#NetTcp) |
-||WebSockets|[:warning:](#transports_websockets) | :white_check_mark: | [:warning:](#transports_websockets) | :grey_question: | :grey_question: | :grey_question: | :grey_question: |
-|Channel types|Request/Reply|:white_check_mark: | :white_check_mark: | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
-||Duplex|:white_check_mark: | :white_check_mark: | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
-|Encodings|Text|:white_check_mark: | :white_check_mark: | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
-||Binary|:white_check_mark: | :white_check_mark: | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
-|TransferMode|Buffered (sync/async)|:white_check_mark: | :white_check_mark: | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
-||Streamed (sync/async)|[:warning:](#transfermode_streamed) | [:warning:](#transfermode_streamed) | [:warning:](#transfermode_streamed) | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
-|MessageVersion|SOAP 1.1 UTF8|:white_check_mark: | :white_check_mark: | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
-||SOAP 1.2 UTF8|:white_check_mark: | :white_check_mark: | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
-|Contracts | DataContract | :white_check_mark: | :white_check_mark: | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
-| | XmlSerializer | :white_check_mark: | :white_check_mark: | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
-| | MessageFormat | :white_check_mark: | :white_check_mark: | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
-| | FaultContract | [:warning:](#contracts_faultcontract) | :white_check_mark: | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
-|Security | None | :white_check_mark: | :white_check_mark: | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
-| | Transport | :white_check_mark: | :white_check_mark:  | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
+|Bindings|BasicHttpBinding|:no_entry_sign: | :white_check_mark: | :white_check_mark: | :grey_question: | :grey_question: | :grey_question: | :white_check_mark: |
```

Comment:
Should all our UWP :no_entry_sign: entries also include links to the section describing why they don't work?